packages/google_maps_flutter/google_maps_flutter_web/lib/src/ground_overlays.dart

```diff
@@ -0,0 +1,108 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of '../google_maps_flutter_web.dart';
+
+/// This class manages all the [GroundOverlayController]s associated to a [GoogleMapController].
+class GroundOverlaysController extends GeometryController {
+  /// Creates a new [GroundOverlaysController] instance.
+  ///
+  /// The [stream] parameter is a required [StreamController] used for
+  /// emitting ground overlay tap events.
+  GroundOverlaysController({
+    required StreamController<MapEvent<Object?>> stream,
+  })  : _streamController = stream,
+        _groundOverlayIdToController =
+            <GroundOverlayId, GroundOverlayController>{};
+
+  final Map<GroundOverlayId, GroundOverlayController>
+      _groundOverlayIdToController;
+
+  // The stream over which polylines broadcast their events
```

Comment:
I don't understand how ground overlay is related to polylines. Does ground overlay use polylines? 

Also, this comment should probably start with `///`

Reply:
Good catch. this is not public object, so public dartdoc format is not used (same thing with other objects)
will change the `polylines` to `ground overlays` here.